site/cypress/e2e/Cypress-TAGrading/clearing_version_conflicts.spec.js

```diff
@@ -0,0 +1,41 @@
+describe('Test cases for checking the clear version conflicts button in the TA grading interface', () => {
+    it('Button should not appear if there are no version conflicts', () => {
+        cy.login('instructor');
+        cy.visit(['sample', 'gradeable', 'grading_homework', 'grading', 'grade?who_id=LHmpGciilVzjcEJ&sort=id&direction=ASC']);
+        cy.get('#grading_rubric_btn').click();
+        cy.get('[data-testid="change-graded-version"]').should('not.exist');
+        cy.get('.version-warning').should('not.exist');
+    });
+
+    it('Button should appear if there are version conflicts', () => {
+        cy.login('instructor');
+        cy.visit(['sample', 'gradeable', 'grading_homework', 'grading', 'grade?who_id=K8jI3q4qpdCc1jw&sort=id&direction=ASC&gradeable_version=1']);
+        cy.get('#grading_rubric_btn').click();
+        cy.get('[data-testid="change-graded-version"]').should('exist');
+        cy.get('.version-warning').should('exist');
```

Comment:
These should still be able to use testids, as they are editable in the Twig files

Reply:
I figured I shouldn't since this PR doesn't touch those elements, should I add them anyway?

Reply:
You can edit elements that need testids, because maybe later another test will use the same elements, and need the testids also. These are small changes, and easy to fix conflicts if any arise.

Reply:
Ok I added the test ids for the elements here: https://github.com/Submitty/Submitty/pull/10689. I'll update this PR's test once that's merged.

Reply:
Fixed in https://github.com/Submitty/Submitty/pull/10635/commits/2e017659349bacf01807dca03e21011458b7b6b2